subworkflows/local/classify_mtdna_haplogroup.nf

```diff
@@ -0,0 +1,28 @@
+// TODO nf-core: If in doubt look at other nf-core/subworkflows to see how we are doing things! :)
+//               https://github.com/nf-core/modules/tree/master/subworkflows
+//               You can also ask for help via your pull request or on the #subworkflows channel on the nf-core Slack workspace:
+//               https://nf-co.re/join
+// TODO nf-core: A subworkflow SHOULD import at least two modules
```

Comment:
Is this true here? We don't really need more than one in this case, but it seems logical to have this as a subworkflow.

(The Y-DNA stuff will definitely offer making a reference genome that can be used by Yleaf as part of a subworkflow.)

Reply:
Importing one one module is fine 👍 
I agree that a SWF makes more sense.